test/gtest/cba.hpp

```diff
@@ -113,17 +116,32 @@ std::vector<ConvTestCase> GetNetwork1()
             {64, 64, 56, 56, 64, 3, 3, 1, 1, 1, 1, 1, 1}};
 }
 
+inline int SetTensorLayout(miopen::TensorDescriptor& desc)
```

Comment:
Perhaps this should be fixed in the tensor class with a constructor that accepts layout as an argument ?

Reply:
I agree! Let me fix this in constructor of [test/tensor_holder.hpp](https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/test/tensor_holder.hpp)